ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/tunnel_interfaces.schema.yml

```diff
@@ -0,0 +1,73 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  tunnel_interfaces:
+    type: list
+    primary_key: name
+    convert_types:
+      - dict
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          description: Tunnel Interface Name
+        description:
+          type: str
+        shutdown:
+          type: bool
+        mtu:
+          type: int
```

Comment:
add 
```yaml
convert_dicts:
- str
```
to all int values, and vice versa if the str value could have a valid value which is just a number.

Reply:
Addressed in 0db9c4a